packages/beacon-node/src/db/repositories/dataColumnSidecarArchive.ts

```diff
@@ -25,21 +24,21 @@ export class DataColumnSidecarArchiveRepository extends PrefixedRepository<Slot,
   }
 
   encodeKeyRaw(prefix: Slot, id: ColumnIndex): Uint8Array {
-    return Buffer.concat([intToBytes(prefix, 4), intToBytes(id, 4)]);
+    return Buffer.concat([encodeNumberForDbKey(prefix, 4), encodeNumberForDbKey(id, 4)]);
   }
 
   decodeKeyRaw(raw: Uint8Array): {prefix: Slot; id: ColumnIndex} {
     return {
-      prefix: bytesToInt(raw.slice(0, 4)) as Slot,
-      id: bytesToInt(raw.slice(4, 8)) as ColumnIndex,
+      prefix: decodeNumberForDbKey(raw, 4) as Slot,
+      id: decodeNumberForDbKey(raw.slice(4), 4) as ColumnIndex,
```

Comment:
using 4 bytes for columns seems too much, right now there are only 128 values of columns. Maybe 2 bytes is enough? then we can support up to 65535 columns with 2 bytes

while using 4 bytes for Slot does not seem enough, I know it's too far to be worried about overflow but we want to make it compliant to our data structure, which is 8 bytes

```typescript
export type Slot = UintNumInf64;
```